tutorials/kernelshap_tabular_penguin.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>Any idea what this warning is about? Could you get rid of it by just updating jupyter and ipywidgets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>The first one can be removed after updating ipywidgets. But the last line is related to scikit-learn, which is used in shap and it cannot be resolved by upgrading scikit-learn.</p>